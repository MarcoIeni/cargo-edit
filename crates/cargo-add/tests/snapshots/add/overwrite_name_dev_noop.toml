bin.name = "cargo-add"
args = ["add", "cargo-list-test-fixture-dependency", "--dev"]
status = "success"
stdout = ""
stderr = """
warning: Unrecognized features: [\"one\", \"two\"]
      Adding cargo-list-test-fixture-dependency (local) to dev-dependencies.
             Features:
             + one
             + two
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
