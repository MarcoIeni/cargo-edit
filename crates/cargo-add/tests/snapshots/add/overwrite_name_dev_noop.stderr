warning: Unrecognized features: ["one", "two"]
      Adding cargo-list-test-fixture-dependency (local) to dev-dependencies.
             Features:
             + one
             + two
